src/lib/logger.ts

```diff
@@ -0,0 +1,7 @@
+export function logAndThrowError(error: unknown, title: string = "Error:") {
```

Comment:
You can use just `title = "Error: "` here, and might want to include the space at the end too. The type of `string` is inferred.

Reply:
The function throws error separating messages with space as default
`throw new Error(`${title} ${error.message}`);`
I think we can skip including space at the end , this would be an additional though